--- usr/i686-w64-mingw32/sys-root/mingw/lib/glib-2.0/include/glibconfig.h.orig	2011-07-15 15:30:33 -0500
+++ usr/i686-w64-mingw32/sys-root/mingw/lib/glib-2.0/include/glibconfig.h	2011-08-19 15:42:52 -0500
@@ -107,9 +107,11 @@
 #ifdef	__cplusplus
 #define	G_HAVE_INLINE	1
 #else	/* !__cplusplus */
+#ifndef _MSC_VER
 #define G_HAVE_INLINE 1
 #define G_HAVE___INLINE 1
 #define G_HAVE___INLINE__ 1
+#endif /* _MSC_VER */
 #endif	/* !__cplusplus */
 
 #ifdef	__cplusplus
