--- demos/gtk-demo/demowindow.cc.orig	2011-06-14 02:02:08 -0500
+++ demos/gtk-demo/demowindow.cc	2011-08-16 23:33:39 -0500
@@ -213,6 +213,8 @@
 
 #ifdef HAVE_GETC_UNLOCKED
       c = getc_unlocked (stream);
+#else
+      c = getc (stream);
 #endif //GLIBMM_PROPERTIES_ENABLED
       if (c == EOF)
 	goto done;
@@ -226,6 +228,8 @@
 	  {
 #ifdef HAVE_GETC_UNLOCKED
 	    int next_c = getc_unlocked (stream);
+#else
+	    int next_c = getc (stream);
 #endif //GLIBMM_PROPERTIES_ENABLED
 	    if (!(next_c == EOF ||
 		  (c == '\r' && next_c == '\n') ||
