diff -burN libxml++.orig/io/istreamparserinputbuffer.cc libxml++/io/istreamparserinputbuffer.cc
--- libxml++.orig/io/istreamparserinputbuffer.cc	2013-09-29 04:45:16.000000000 -0500
+++ libxml++/io/istreamparserinputbuffer.cc	2015-04-25 23:21:46.392578500 -0500
@@ -39,6 +39,6 @@
 
   bool IStreamParserInputBuffer::do_close()
   {
-    return input_;
+    return !input_.fail();
   }
 }
diff -burN libxml++.orig/io/ostreamoutputbuffer.cc libxml++/io/ostreamoutputbuffer.cc
--- libxml++.orig/io/ostreamoutputbuffer.cc	2013-09-29 04:45:16.000000000 -0500
+++ libxml++/io/ostreamoutputbuffer.cc	2015-04-25 23:24:57.258269000 -0500
@@ -29,13 +29,13 @@
     // here we rely on the ostream implicit conversion to boolean, to know if the stream can be used and/or if the write succeded.
     if(output_)
       output_.write(buffer, len);
-    return output_;
+    return !output_.fail();
   }
 
   bool OStreamOutputBuffer::do_close()
   {
     if(output_)
         output_.flush();
-    return output_;
+    return !output_.fail();
   }
 }
